crates/rattler_conda_types/src/package/package_metadata.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+/// This is metadata about the package version that is contained in `.conda` packages only, in the
+/// outer zip archive.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct PackageMetadata {
```

Comment:
Maybe you can add an implementation of `Default` that constructs this with the value 2?